pipeline_dp/pipeline_backend.py

```diff
@@ -132,6 +132,20 @@ def flatten(self, col1, col2, stage_name: str):
         """
         pass
 
+    def annotate(self, col, stage_name: str, **kwargs):
+        """Annotates collection with registered annotators.
+
+        Args:
+          col: input collection which contains tuples (key, accumulator).
+          stage_name: name of the stage.
+          kwargs: additional key-value pair arguments supported by
```

Comment:
Nit: no need to mention DPEngine.aggregate, there is another functiton which need to be annotated (`DPEngine.select_partitions`) and there might be more.

Nit: `kwargs` additional arguments about the current aggregation.